controls/cis_al2023.yml

```diff
@@ -0,0 +1,2166 @@
+
+  - id: 1.1.5.1
+    title: Ensure separate partition exists for /var/log (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - partition_for_var_log
+
+  - id: 1.1.5.2
+    title: Ensure nodev option set on /var/log partition (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - mount_option_var_log_nodev
+
+  - id: 1.1.5.3
+    title: Ensure noexec option set on /var/log partition (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - mount_option_var_log_noexec
+
+  - id: 1.1.5.4
+    title: Ensure nosuid option set on /var/log partition (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - mount_option_var_log_nosuid
+
+  - id: 1.1.6.1
+    title: Ensure separate partition exists for /var/log/audit (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - partition_for_var_log_audit
+
+  - id: 1.1.6.2
+    title: Ensure noexec option set on /var/log/audit partition (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - mount_option_var_log_audit_noexec
+
+  - id: 1.1.6.3
+    title: Ensure nodev option set on /var/log/audit partition (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - mount_option_var_log_audit_nodev
+
+  - id: 1.1.6.4
+    title: Ensure nosuid option set on /var/log/audit partition (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - mount_option_var_log_audit_nosuid
+
+  - id: 1.1.7.1
+    title: Ensure separate partition exists for /home (Automated)
+    levels:
+      - l2_server
+    status: automated
+    rules:
+      - partition_for_home
+
+  - id: 1.1.7.2
+    title: Ensure nodev option set on /home partition (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - mount_option_home_nodev
+
+  - id: 1.1.7.3
+    title: Ensure nosuid option set on /home partition (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - mount_option_home_nosuid
+
+  - id: 1.1.8.1
+    title: Ensure /dev/shm is a separate partition (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - partition_for_dev_shm
+
+  - id: 1.1.8.2
+    title: Ensure nodev option set on /dev/shm partition (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - mount_option_dev_shm_nodev
+
+  - id: 1.1.8.3
+    title: Ensure noexec option set on /dev/shm partition (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - mount_option_dev_shm_noexec
+
+  - id: 1.1.8.4
+    title: Ensure nosuid option set on /dev/shm partition (Automated)
+    levels:
+      - l1_server
+    status: automated
+    rules:
+      - mount_option_dev_shm_nosuid
+
+  - id: 1.1.9
+    title: Ensure usb-storage is disabled (Automated)
```

Comment:
Replace with "title: Disable USB Storage (Automated)".

Reply:
both CIS WorkBench (online) and its related PDF show me: "Ensure usb-storage is disabled" . 
I am looking at latest official version made by Eric Pinnell ( CIS Amazon Linux 2023 Benchmark v1.0.0 ).  

Am I looking at the wrong place?

Reply:
I'm referring to `CIS_Amazon_Linux_2023_Benchmark_v1.0.0.pdf` available on https://downloads.cisecurity.org/. If there are multiple versions of this file, this is going to be complicated :slightly_smiling_face:. Do you have a link to the version you use?